pipeline/upstream_test_executor.groovy

```diff
@@ -22,14 +22,14 @@ def failureReason
 def reportPotalLink
 
 // Pipeline script entry point
-node('ceph-qe-ci || rhel-8-medium') {
+node('ceph-qe-ci || rhel-9-medium') {
     try {
         stage('Preparing') {
             if (env.WORKSPACE) { sh script: "sudo rm -rf * .venv" }
             checkout(
                 scm: [
                     $class: 'GitSCM',
-                    branches: [[name: 'origin/master']],
+                    branches: [[name: 'origin/fix-upstream']],
```

Comment:
@sunilkumarn417 , "branch and git repo "changes is this intentional??

Reply:
Just for testing.

Reply:
Execution is done now, will revert back to right repo and branch.